docs/source/tutorial.md

```diff
@@ -0,0 +1,168 @@
+     - python=3.10
+     - notebook
+     - r-essentials
+     - r-plotly
+     - r-shiny
+   platforms:
+     - osx-arm64
+     - linux-64
+   ```
+
+   ```{note}
+   Refer to [conda-project create](user_guide) for a detailed list of options.
+   ```
+
+5. Investigate the contents of the newly created conda project file:
+   ```yaml
+   $ cat conda-project.yml
+   name: learn-cp
+     environments:
+   default:
+     - environment.yml
+   variables: {}
+   commands: {}
+   ```
+
+   Note that the `name` key is populated with the value supplied in the `create` action.
+
+   An upcoming section of the tutorial will involve adding a custom command to the `commands` key. It's also
+   possible to define variables using the `variables` key.  These key varlue pairs are loaded as overrides
+   to the inherited execution environment when a `run` action is issued.
+
+6. Initiate the `prepare` action on the project:
+   ```shell
+   $ conda-project prepare
+   ```
+
+   ```{note}
+   As noted in a [preceding section](user_guide), the `prepare` action will initiate conda install
+   for the active environment.
+   ```
+
+7. Activate the project's default environment by calling the `activate` action:
+   ```shell
+   $ conda-project activate
+   ```
+
+## Create an example notebook-based shiny app
+
+In this section of the tutorial, Jupyter notebook will be used to develop and refine the shiny app.
+
+1. Use `conda-project` to launch Jupyer notebook.
+   ```shell
+   $ conda-project run jupyter notebook
+   ```
+
+2. Start a `R` kernel by clicking *New* and select the option labeled **R**.
+
+3. Paste the following content into the first cell of the notebook and then run the cell.
+   ```r
+   library(shiny)
+   library(data.table)
+   library(ggplot2)
+
+   ui <- fluidPage(
+       titlePanel('Auto MPG'),
+       sidebarPanel(
+           radioButtons("origin", h3("Origin"),
+                       choices = list("Asia" = "Asia"),
+                       selected = "Asia")
+       ),
+       mainPanel(plotOutput(outputId = "distPlot"))
+   )
+
+   server <- function(input, output) {
+       df = fread('http://bit.ly/autompg-csv')
+       output$distPlot <- renderPlot({
+           ggplot(data = df[origin==input$origin], mapping = aes(mpg)) + geom_density()
+       })
+   }
+
+   shinyApp(ui = ui, server = server)
+   ```
+
+4. A link will appear at the bottom of the cell once the shiny app is being served.  Click the link and
+   verify the output.
+
+5. With the first cell selected, click the Stop button to stop the server.
+
+6. Modify the content in the first cell by adding the US and Europe to the choices list in the UI:
+
+   ```R
+        choices = list("Asia" = "Asia", "Europe" = "Europe", "US" = "US"),
+   ```
+
+7. Run the cell again. When the server link appears, navigate to the shiny App and verify the code update is
+   reflected in the UI.
+
+8. Finally, save the verified R code to a file called `app.R` in the project directory.
```

Comment:
I haven't looked a R in Jupyter notebooks in a while. Can the notebook be converted to an R script from the File -> download menu?

Reply:
I'll check that. That last step is annoying and would be much better in the R session.